stats/opentelemetry/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	go.opentelemetry.io/otel/metric v1.28.0
 	go.opentelemetry.io/otel/sdk v1.28.0
 	go.opentelemetry.io/otel/sdk/metric v1.28.0
-	google.golang.org/grpc v1.65.0
```

Comment:
Shouldn't we use 1.66.0?

Reply:
Done, 1.66.0 was not available at the time.